sparkasse-adapter/src/main/java/de/adorsys/xs2a/gateway/service/provider/SparkasseAccountInformationServiceProvider.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2018-2018 adorsys GmbH & Co KG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package de.adorsys.xs2a.gateway.service.provider;
+
+import de.adorsys.xs2a.gateway.service.ais.AccountInformationService;
+import de.adorsys.xs2a.gateway.service.impl.SparkasseAccountInformationService;
+
+public class SparkasseAccountInformationServiceProvider implements AccountInformationServiceProvider {
+
+    private SparkasseAccountInformationService accountInformationService;
+
+    @Override
+    public String getBankCode() {
+        return "99999999";
```

Comment:
Seems it is not real bank code of Sparkasse bank. Quick search shows this number `79050000`

Reply:
IBANs used in their sandbox have this bank code.
Where did you found '79050000'? I got very different result https://www.bankleitzahlen.ws/Sparkasse